src/translate/expr.rs

```diff
@@ -113,7 +113,7 @@ pub fn translate_option_expr(sql_option_expr: &Option<Box<SqlExpr>>) -> Result<O
     match sql_option_expr {
         Some(expr) => match translate_expr(expr).map(Box::new) {
             Ok(expr) => Ok(Some(expr)),
-            Err(_) => Err(TranslateError::UnsupportedExpr(expr.to_string()).into()),
+            Err(_) => unreachable!(),
```

Comment:
Here I change this part to unreachable, because `sqlparser-rs`'s `ParseError` will never let the program to reach that branch.